lib/open_pantry/web/views/display_logic.ex

```diff
@@ -12,6 +12,29 @@ defmodule OpenPantry.Web.DisplayLogic do
     end
   end
 
+  @doc """
+  Removes spaces from a string, converts it to "underscore" format,
+  and replaces the underscores with dashes.  Used mainly for types of
+  stock or credit.  For more on the underscore format, see:
+  https://hexdocs.pm/elixir/Macro.html#underscore/1
+
+  This does not handle acronyms well when not followed by a capital letter,
+  with maybe spaces between.  See examples.
+
+  ## Examples:
+
+  iex> OpenPantry.Web.DisplayLogic.dasherize "American Express"
+  "american-express"
+
+  iex> OpenPantry.Web.DisplayLogic.dasherize "Foo.Bar"
+  "foo/bar"
+
+  iex> OpenPantry.Web.DisplayLogic.dasherize "NASA Low Residue"
+  "nasa-low-residue"
+
+  iex> OpenPantry.Web.DisplayLogic.dasherize "NASA-Approved Low Residue"
+  "nas-a--approved-low-residue"
```

Comment:
This seems like a bug and/or a very weird example...

Reply:
I saw comment about this I guess in your explanation and appreciate effort to draw out limitations of what it does at present, but not sure it makes sense to highlight this as "expected behavior" since its unlikely this is ever what anyone wants or expects from this input....

Reply:
Agreed, it is rather weird.  Removed for simplicity.